docs/examples/buildConfigs/README.md

```diff
@@ -0,0 +1,28 @@
+# BuildConfig
+
+A build configuration describes a single build definition and a set of triggers for when a new build should be created. A build configuration is defined by a BuildConfig.
+
+In Kedge, defining BuildConfigs has been implemented by merging `BuildConfigSpec` and `ObjectMeta` at the root YAML level of the spec.
+
+A BuildConfig can be defined in Kedge as follows -
+
+```yaml
+buildConfigs:
+- triggers:
+  - type: "ImageChange"
+  source:
+    type: "Git"
+    git:
+      uri: "https://github.com/openshift/ruby-hello-world"
+    dockerfile: "FROM openshift/ruby-22-centos7\nUSER example"
```

Comment:
how is this going to work, when there is `docekrfile` is specified, but build strategy is `sourceStrategy` (S2I)?

This doesn't look good to me.

Reply:
I've tried that and `dockerfile` is being ignored

Reply:
```console
$ oc explain bc.spec.source.dockerfile
FIELD: dockerfile <string>

DESCRIPTION:
     dockerfile is the raw contents of a Dockerfile which should be built. When
     this option is specified, the FROM may be modified based on your strategy
     base image and additional ENV stanzas from your strategy environment will be
     added after the FROM, but before the rest of your Dockerfile stanzas. The
     Dockerfile source type may be used with other options like git - in those
     cases the Git repo will have any innate Dockerfile replaced in the context
     dir.
```

Reply:
I saw that @containscafeine  ;-), But don't understand how this can work with S2I. S2I is not using Dockerfile does it?